src/client/webpack.config.js

```diff
@@ -131,7 +131,7 @@ module.exports = {
     new webpack.NoEmitOnErrorsPlugin(),
     new webpack.DefinePlugin({
       'process.env': {
-        NODE_ENV: JSON.stringify('production'),
+        APP_ENV: JSON.stringify('production'),
```

Comment:
@SobakaSlava it doesn't seem right, APP_ENV here shouldn't be equal to production by default. It should take an actual `APP_ENV` from config instead  (ex. for staging environment it should have APP_ENV==='staging')

Reply:
this config is used only for non dev purposes. for dev purposes `webpack.dev.config` is used

Reply:
@SobakaSlava 
But what about staging environment? `APP_ENV` should be `staging` in that case (and `webpack.config.js` used)